yuniql-platforms/oracle/SchemaUpgrade_v1_1.sql

```diff
@@ -0,0 +1,25 @@
+﻿CREATE TABLE ${YUNIQL_DB_NAME}.${YUNIQL_TABLE_NAME}_v1_0 AS SELECT * FROM ${YUNIQL_DB_NAME}.__yuniqldbversion;
```

Comment:
We can delete this file because its a new platform and by default will use v1.1 schema